client/wp-admin-scripts/print-shipping-label-banner/shipping-banner/index.js

```diff
@@ -382,13 +382,18 @@ export class ShippingBanner extends Component {
 											target="_blank"
 											type="external"
 											onClick={
-												this.woocommerceServiceLinkClicked
+												this
```

Comment:
Stray newline?

Reply:
This is weird. I think it maybe the auto commit bot doing. Let me check.

Reply:
Yep, seems to be from prettify.